src/utils/validation.ts

```diff
@@ -0,0 +1,8 @@
+function isValidEmail(email: string) {
+  if (/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(.\w{2,3})+$/.test(email)) {
+    return true
+  }
+  return false
```

Comment:
Simplifies to 

```suggestion
  return (/^\w+([.-]?\w+)*@\w+([.-]?\w+)*(.\w{2,3})+$/.test(email)
```

We'll need a regex that includes the username (X), domain (Y) and extension(Z) -- X@Y.Z. This one accepts X@Y as valid (eg. try ajith@newfang).

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/121/commits/7b00c57f2704f07bd58178b721d4f56fff015592